browser/src/core/Socket.js

```diff
@@ -1387,7 +1387,13 @@ app.definitions.Socket = L.Class.extend({
 		else if (this._reconnecting) {
 			// we are reconnecting ...
 			this._map._docLayer._refreshTilesInBackground();
-			this._map.fire('statusindicator', {statusType: 'reconnected'});
+			this._map.fire('statusindicator', { statusType: 'reconnected' });
+
+			var selectedMode = this._map.uiManager.getDarkModeState();
+			var cmd = { 'NewTheme': { 'type': 'string', 'value': '' } };
+			selectedMode ? cmd.NewTheme.value = 'Dark' : cmd.NewTheme.value = 'Light';
+			app.socket.sendMessage('uno .uno:ChangeTheme ' + JSON.stringify(cmd));
```

Comment:
This also could be inside some helper function and be used in Control.UIManager.js as well...
like `activateDarkMode(true / false)` .  You can put it in UIManager.

and be used in:` toggleDarkMode` https://github.com/CollaboraOnline/online/blob/bb246318fef5b409c237dc7bf34147a164c18b0d/browser/src/control/Control.UIManager.js#L118

and in `initDarkModeFromSettings` https://github.com/CollaboraOnline/online/blob/bb246318fef5b409c237dc7bf34147a164c18b0d/browser/src/control/Control.UIManager.js#L147